<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomFrame</class>
 <widget class="QFrame" name="RoomFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>228</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="roomsSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ChatWidget" name="outputBrowser">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListView" name="usersView">
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="InputEdit" name="inputEdit">
     <property name="maxLength">
      <number>1024</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QTextBrowser</extends>
   <header>chatwidget.h</header>
  </customwidget>
  <customwidget>
   <class>InputEdit</class>
   <extends>QLineEdit</extends>
   <header>inputedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
